lib/ngx/balancer.lua

```diff
@@ -260,4 +273,19 @@ if subsystem == 'http' then
 end
 
 
+function _M.set_proxy_bind(addr)
```

Comment:
please add test cases to cover this new API.

Reply:
sorry, i did't know how to add test cases for new `set_proxy_bind` API, because it depends https://github.com/openresty/stream-lua-nginx-module/pull/350 and https://github.com/openresty/lua-nginx-module/pull/2324

Reply:
https://github.com/openresty/lua-resty-core/blob/master/.travis.yml#L82
https://github.com/openresty/lua-resty-core/blob/master/.travis.yml#L73

change the repo to your repo: eg https://github.com/ytlm/lua-nginx-module/tree/feature/balancer_proxy_bind

Reply:
If there are not test cases, I can't merge this PR.

Reply:
Please apply the following patch.
And add another test file for the stream module.

```
commit 2eaf5fc44083f3a0dfeff71a296302a7d26e079b
Author: lijunlong <lijunlong@openresty.com>
Date:   Fri Jun 28 12:12:36 2024 +0800

    fixed tests.

diff --git a/t/upstream-balancer-set-proxy-bind.t b/t/upstream-balancer-set-proxy-bind.t
index 362bfae..9357ca8 100644
--- a/t/upstream-balancer-set-proxy-bind.t
+++ b/t/upstream-balancer-set-proxy-bind.t
@@ -9,7 +9,7 @@ use t::TestCore;
 
 repeat_each(2);
 
-plan tests => repeat_each() * (blocks() * 4 + 6);
+plan tests => repeat_each() * (blocks() * 4);
 
 $ENV{TEST_NGINX_LUA_PACKAGE_PATH} = "$t::TestCore::lua_package_path";
 
@@ -52,6 +52,8 @@ qr/127.0.0.1/,
 [error]
 [warn]
 
+
+
 === TEST 2: balancer with set_proxy_bind (addr)
 --- http_config
     lua_package_path "$TEST_NGINX_LUA_PACKAGE_PATH";
@@ -86,6 +88,8 @@ qr/127.0.0.4/,
 [error]
 [warn]
 
+
+
 === TEST 3: balancer with set_proxy_bind (addr and port)
 --- http_config
     lua_package_path "$TEST_NGINX_LUA_PACKAGE_PATH";
```

Reply:
thank you very much.